config/crd/bases/ndb.nutanix.com_databases.yaml

```diff
@@ -56,19 +56,14 @@ spec:
                       database instance (password and ssh key)
                     type: string
                   databaseInstanceName:
-                    default: database_instance_name
-                    description: Name of the database instance
+                    description: Name of the database instance, default "database_instance_name"
```

Comment:
Since we're not defaulting anymore, can we update this? 
Maybe try running make generate manifests to see if this gets auto generated?

Reply:
Yes this gets auto generated. running make generate fixed this. I think we can make a single target to be run before commiting to git. eg make commit, which will run all targets in one run.